framework/validators/AtLeastOneValidator.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\validators;
+
+use Yii;
+use yii\base\InvalidConfigException;
+
+/**
+ * AtLeastValidator checks if at least one of many attributes are filled.
+ *
```

Comment:
AtLeastValidator? or AtLeastOneValidator? and is it about 'requiring?'....checking is a bit vague here

Reply:
I am OK with this naming. It is simple enough

Reply:
unfortunately it is ambiguous. AtLeast could also point to a Validator which checks if a value is `at least` a certain value....